packages/web-components/@sfx/ui/test/list.test.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from './utils';
+import List from '../src/list';
+import { Base } from '@sfx/base';
+import { TemplateResult } from 'lit-element';
+
+describe('List component', () => {
+  it('should extend Base', () => {
+    const list = new List();
+    expect(list).to.be.an.instanceof(Base);
+  });
+
+  let list: any = {};
```

Comment:
No type or initialization is necessary.
```suggestion
  let list;
```